src/emu/epoc/include/epoc/services/akn/skin/skn.h

```diff
@@ -240,6 +241,25 @@ namespace eka2l1::epoc {
         skn_attrib_info attrib;
     };
 
+    struct skn_layout_info {
+        std::int32_t layout_type;
+        vec2 layout_size;
+    };
+
+    struct skn_scalable_item_def {
```

Comment:
This struct belongs in chunk_maintainer.cpp. You should throw it in there. There are some def structs that are similar to this at the beginning of that file.

Reply:
This file is like, the skin file parser.